### PR TITLE
refactor main file

### PR DESCRIPTION
**Pull request by Adam-Goldman7**
I prefer to keep the main file / entry point as lean as possible

Thoughts?

### CODE REVIEW

File: client/material-ui-wrapped-provider.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+const muiTheme = getMuiTheme({
+  fontFamily: `'Source Sans Pro', 'Helvetica Neue', arial, sans-serif`,
+  palette: {
+    primary1Color: '#6691C2',
+    accent1Color: deepOrange500
+  }
+});
+
+export default class WrappedMuiThemeProvider extends Component {
```
Comment (tomusdrw) [inline client/material-ui-wrapped-provider.js]: It's just a `component` - I would move that to `/components`

File: client/middlewares.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import Web3 from 'web3';
```
Comment (tomusdrw) [inline client/middlewares.js]: Could also be `middlewares/index.js`

File: client/routes.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+
```
Comment (tomusdrw) [inline client/routes.js]: This on is also `Component` albeit special, so I'm fine with keeping it here.

File: client/middlewares.js (hunk 2 above)
Comment (Adam-Goldman7) [inline client/middlewares.js]: maybe we put it directly in the existing middleware/index.js?

it imports everything from there anyhow

Comment (tomusdrw) [inline client/middlewares.js]: Yeah, why not.

**Pull request merged**